src/cartridges/bm_adyen/cartridge/static/default/adyenSettings.js

```diff
@@ -51,6 +52,42 @@ document.addEventListener('DOMContentLoaded', () => {
     settingChanged(name, value);
   });
 
+  // add event listener to test connection based on current form contents
+  testConnectionButton.addEventListener('click', async () => {
+    const response = await fetch('AdyenSettings-TestConnection', {
+      settings: changedSettings,
+    });
+    const data = await response.json();
+    if (data.success) {
+      const alertBar = document.getElementById('saveChangesAlert');
```

Comment:
Shouldn't we just show a checkmark instead of the saveChangesAlert when the TestConnection is successful?

Reply:
I don't know. the design does not really show that.